harvester/gene_enricher.py

```diff
@@ -35,20 +35,23 @@
 data = None
 
 
-def get_genes(identifier):
+def get_gene(identifier):
     """ return gene for identifier """
     for store in [GENES, ALIASES]:
         genes = store.get(identifier, None)
-        if genes:
+        if genes and len(genes) == 1:
             return genes
```

Comment:
I know this is already merged, but perhaps `return genes[0]` here better fits the `get_gene` name for this function.